src/services/namex-services.ts

```diff
@@ -141,7 +141,8 @@ export default class NamexServices {
    */
   static async searchColin (corpNum: string): Promise<any> {
     // remove BC prefix as COLIN only supports base number with no prefix for BC's
-    const url = `${appBaseURL}/colin/${corpNum}`
+    const cleanedCorpNum = corpNum.replace(/^BC+/i, '')
+    const url = `${appBaseURL}/colin/${cleanedCorpNum}`
```

Comment:
Does this only search by corp num, or also by name?

Is it OK to let through ids like CPxxx or FMxxx, etc?

Reply:
search only with corp num. looks like only for BC prefix we need to search with number

Reply:
Oh, right. This is called when a user enters a business id to find (for name change, alteration, etc). All good 👍